docs/reference/services.rst

```diff
@@ -0,0 +1,63 @@
+=================
+MicroOVN Services
+=================
+
+This document provides brief explanation of services provided by MicroOVN. They
+are generally not meant for direct user interaction and should be used only
+if necessary. List and status of these services can be displayed by running:
+
+.. code-block:: none
+
+    snap services microovn
+
+microovn.central
+----------------
+
+.. warning::
+
+    Deprecated: This service will be removed in future release
+
+This is a shortcut service. Starting this service will start and enable
+``microovn.ovn-nb``, ``microovn.ovn-sb`` and ``microovn.ovn-northd``. However
+this service , due to the ``snap`` limitations, is not capable of stopping
+child services and its usage is generally discouraged.
```

Comment:
I'd rather highlight the reasons for why managing these services separately is desirable rather than bashing snaps here

Reply:
Sure, I'd angle it with a message telling the user to use the other services to stop things instead then. Together with the transitional label it will make it absolutely clear to the user that they should stop whatever it is they think they should be doing and follow the new workflow